geometries/GeoDriver_AvoidTest.py

```diff
@@ -1,66 +0,0 @@
-#from altitude.AsperaAltitude import altitude
-from beta.beta_Algorithm import beta
-#from eclipse.AsperaEclipse import eclipse
-from geometries.latlon.AsperaLatLon_Algorithm import latlon
-
-import spiceypy as sp
-import spiceypy.utils.support_types as stypes
-
-import csv
-import pytest
```

Comment:
This script used artifact data for HST, and HST data was removed from test kernel set, so this script has been removed.